katdal/applycal.py

```diff
@@ -379,7 +379,7 @@ def indirect_cal_product(cache, name, product_type):
         if not timestamps:
             raise KeyError("No cal product '{}' parts found (expected {})"
                            .format(name, n_parts))
-        return SimpleSensorData(indirect_cal_product_name(name, product_type),
+        return SimpleSensorGetter(indirect_cal_product_name(name, product_type),
                                 np.array(timestamps), np.array(values))
```

Comment:
It looks like the indentation is misaligned here.

Reply:
Thanks, fixed (and if you had a `.flake8` then Jenkins would have shouted at me - it looks like katdal is pretty close to being flake8-clean so shouldn't be much work).